README.md

```diff
@@ -46,6 +46,18 @@ tr.request(
   }
 )
 
+// Or use tr.request as a promise
+const res = await tr.request(
+  {
+    method: 'session-get',
+    arguments: {
+      fields: ['version'],
+    },
+  }
+)
+
+console.log(res)
+
 tr.close()
```

Comment:
Maybe we should start the doc with the Promise example? as it's the defacto standard.

Reply:
Yep we sould open a PR for `react-native-transmission` & `transmission-native`